tests/test_decision_maker/test_base.py

```diff
@@ -826,7 +597,6 @@ def test_is_affordable_ledger_state_proxy(self):
                 self.decision_maker, "_settle_tx", return_value="tx_digest"
             ):
                 self.decision_maker._is_affordable(tx_message)
-                assert not self.decision_maker.message_out_queue.empty()
```

Comment:
why is that assert removed?

Reply:
You could also add here: `self.decision_maker.message_out_queue.get()`

Reply:
That assert was simply wrong. In the test there's no way the decision maker can produce an outgoing message.

Reply:
It worked so far because of interdependency with other tests.